package.json

```diff
@@ -52,6 +50,7 @@
     "@types/react-test-renderer": "^16.9.3",
     "@typescript-eslint/eslint-plugin": "^3.7.1",
     "@typescript-eslint/parser": "^3.7.1",
+    "fast-deep-equal": "^3.1.3",
```

Comment:
Should this still be here? It seems like the import is removed in this change.

Reply:
No, we don't need it here. I was assuming we might have examples in this repo, but all examples are on codesandbox, so we can remove it.